runtime/cudaq/platform/default/rest/helpers/ionq/ionq.config

```
@@ -49,6 +51,14 @@ while [ $# -gt 1 ]; do
 		PLATFORM_EXTRA_ARGS="$PLATFORM_EXTRA_ARGS;noise;$2"
 		IONQ_NOISE_MODEL="$2"
 		;;
+	--error-mitigation)
```

Per the comment in lines 40-41, this command line option needs to start with `ionq`
```suggestion
	--ionq-error-mitigation)
```

---

ah i see! thanks